src/stitch_fix/generateId.js

```diff
@@ -1,6 +1,8 @@
 import { ulid } from 'ulid';
 import { constantCase } from 'constant-case';
 
+// Keep the type signature of this function in sync with the types exposed in
+// index.d.ts
```

Comment:
this is actually now in `generateId.d.ts`. i would also add a companion comment in `generateId.d.ts` to mention that it has to be kept in sync w/ the source file

Reply:
Yep! Done.